gui/resources.xml

```diff
@@ -1234,19 +1272,11 @@ Vocabulary
       <!-- {{{ Layers panel -->
       <child>
         <object class="GtkToggleAction" id="ToggleLayersTool">
-          <property name="label" translatable="yes" context="Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Toggle the Layers dockpanel (arrange layers, or assign modes).</property>
+          <property name="label" translatable="yes" context="Menu→Layers (lables), Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Dockable panel for managing layers.</property>
           <property name="icon-name">mypaint-layers-symbolic</property>
           <signal name="activate" handler="toggle_dockpanel_cb"/>
         </object>
-      </child>
-      <child>
-        <object class="GtkAction" id="RevealLayersTool">
-          <property name="label" translatable="yes" context="Menu→Layers (labels), Accel Editor (labels)">Show Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Reveal the Layers dockpanel (arrange layers, or assign modes).</property>
-          <property name="icon-name">mypaint-layers-symbolic</property>
-          <signal name="activate" handler="reveal_dockpanel_cb"/>
-        </object>
```

Comment:
:-1: :-1: :-1:
I really want to keep the reveal commands, because replacing them with toggle commands changes the sidebar order. See #352 for the background to this.

If they are not defined, things will break or code will become dead and thus a burden to maintain.

It may be fine to move this one to